src/app/views/home/home.component.scss

```diff
@@ -158,6 +190,47 @@
   }
 }
 
+.home-jobs-applied-for-header-wrapper {
+  background: $color-gray-25;
+  border-bottom: 2px solid $color-primary-cerise;
+}
+
+.home-jobs-applied-for-header {
+  h3 {
+    margin-top: 15px;
+    margin-bottom: 15px;
+  }
+}
+
+.home-jobs-applied-for-job-item-wapper {
+  border-top: 1px solid $color-gray-25;
+  border-bottom: 1px solid $color-gray-25;
+}
+
+.home-jobs-applied-for-job-item {
```

Comment:
Please use nested structure of scss everywhere. It is more readable and exposes the html structure.